cloudify_cli/cli/helptexts.py

```diff
@@ -222,7 +222,14 @@
                        'If the certificates metadata (I.E: the Manager IP ' \
                        'address) from the snapshot does not match the ' \
                        'Manager metadata, the certificates cannot work on ' \
-                       'this Manager and will not be restored'
+                       'this Manager and will not be restored. In case the ' \
+                       'certificates has been restored, the Manager will be ' \
+                       'automatically rebooted at the end of the execution. ' \
+                       'To avoid automatic reboot, use the flag ' \
+                       '`--no-reboot` (not recommended)'
+NO_REBOOT = 'Do not perform an automatic reboot to the Manager VM after ' \
+            'restoring certificates from snapshot (not recommended). ' \
+            'Only relevant if the flag `--restore-certificates` was supplied'
```

Comment:
restoring certificates from **a** snapshot (not recommended). ' 
Only relevant if the ~flag~ `--restore-certificates` **flag** was supplied.'